routes/contact-router.js

```diff
@@ -0,0 +1,38 @@
+const router = require('express').Router();
+const nodemailer = require('nodemailer');
+
+router.post('/', (req, res) => {
+  let { email, name, message } = req.body;
+  message = message
+    .split('\n')
+    .map((p) => `<p>${p}</p>`)
+    .join('');
+
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: 'tidyhivelife@gmail.com', //replace with your email
+      pass: '123qwe++', //replace with your password
```

Comment:
This should be a Environment Variable?
```suggestion
      pass: '123qwe++', //replace with your password
```